src/components/ha-camera-stream.ts

```diff
@@ -194,11 +235,16 @@ class HaCameraStream extends LitElement {
     fireEvent(this, "iron-resize");
   }
 
-  private _destroyPolyfill(): void {
+  private async _destroyPolyfill(): Promise<void> {
```

Comment:
why is this a async function?

Reply:
Good catch. I made it async at first because I used sendMessage instead of fireMessage, but forgot to change it back.